content/android/geolocation.md

```diff
@@ -0,0 +1,60 @@
+Skygear supports saving Geo Location as a field in record. Please be reminded
+that Geo Location is identified by latitude and longitude.
+
+### Saving a location in a record
+
+```java
+Location loc = /* get the location via LocationServices or create a new one */
```

Comment:
> Can we have a sample location here?

Do you mean I directly create a `Location` instance by latitude and longitude ?

> Is this Location class https://developer.android.com/reference/android/location/Location.html ?

Yes.